packages/react/src/hooks/use-scrollable.ts

```diff
@@ -0,0 +1,68 @@
+import { RefObject, useEffect } from 'react';
+
+interface UseScrollableOptions {
+    scrollableElement: RefObject<HTMLElement>;
+    scrollByPercent: number;
+    onScroll: (params: { atStartX: boolean; atEndX: boolean; }) => void;
+}
+
+interface UseScrollableReturns {
+    scrollToLeft: () => void;
+    scrollToRight: () => void;
```

Comment:
C'est une préférence personnelle
```suggestion
    scrollToLeft(): void;
    scrollToRight(): void;
```